elements/designer-element-palette/designer-element-palette.html

```diff
@@ -64,10 +59,16 @@
     <h1>Elements</h1>
     <div id="elements">
       <template is="dom-repeat" items="{{builtIns}}">
-        <div class="element" on-mousedown="_onMouseDown">&lt;<span>{{item}}</span>&gt;</div>
+        <div class="element" on-mousedown="_onMouseDown">
+          &lt;<span>{{item}}</span>&gt;
+        </div>
       </template>
     </div>
-    <div id="proxy"></div>
+    <div id="proxy"
+        on-mouseover="_onProxyMouseOver"
+        on-mouseout="_onProxyMouseOut"
+        on-mouseup="_onProxyMouseUp">
```

Comment:
https://github.com/Polymer/polymer/blob/master/PRIMER.md#gesture-events

> track - moving while finger/button is down
>   - state - a string indicating the tracking state:
>     - start - fired when tracking is first detected (finger/button down and moved past a pre-set distance threshold)
>     - track - fired while tracking
>     - end - fired when tracking ends
>   - x - clientX coordinate for event
>   - y - clientY coordinate for event
>   - dx - change in pixels horizontally since the first track event
>   - dy - change in pixels vertically since the first track event
>   - ddx - change in pixels horizontally since last track event
>   - ddy - change in pixels vertically since last track event